ansible/roles/test/files/ptftests/inner_hash_test.py

```diff
@@ -88,14 +89,16 @@ def setUp(self):
 
         self.balancing_range = self.test_params.get('balancing_range', self.DEFAULT_BALANCING_RANGE)
         self.balancing_test_times = self.test_params.get('balancing_test_times', self.BALANCING_TEST_TIMES)
+        self.outer_encap_formats = self.test_params.get('outer_encap_formats', self.OUTER_ENCAP_FORMATS)
```

Comment:
We seem to be defining OUTER_ENCAP_FORMATS in both this file as well as conftest.py, further we are taking outer_encap_formats as a param from the ptfrunner caller, but not really passing it down. Can we clean up this behavior, and either pass it down in ptfrunner and define it at caller, or define it here, not doing both

Reply:
done